scanapp/tests/test_models.py

```diff
@@ -51,9 +51,8 @@ def test_scan_added(self):
             scan_start_time=timezone.now(),
             scan_end_time=timezone.now()
         )
-        
-        self.assertEqual('2.0.0rc3', scan.scancode_version)
-        self.assertEqual(str(scan.url), str(scan))
+        self.assertEqual('2.0.0rc3', Scan.objects.get(pk=scan.pk).scancode_version)
+        self.assertEqual(str(Scan.objects.get(pk=scan.pk).url), str(scan))
```

Comment:
I am trying to test the string representation of the models. Isn't it right to test those things?